.travis.yml

```diff
@@ -6,10 +6,10 @@ virtualenv:
 before_install:
   - sudo sh -c "echo deb http://www.openfoam.org/download/ubuntu precise main > /etc/apt/sources.list.d/openfoam.list"
   - sudo apt-get update -qq
-  - sudo apt-get install -y --force-yes openfoam222
+  - sudo apt-get install -y --force-yes openfoam230
```

Comment:
This PR should work with OpenFoam 222 and 230, right?  If so maybe, we can use a build matrix to test using 222 and 230.